python/lsst/cp/pipe/linearity.py

```diff
@@ -453,6 +453,15 @@ def run(self, inputPtc, dummy, camera, inputDims, inputPhotodiodeData=None,
                     binCenters = binCenters[numPerBin > 0]
 
                 self.debugFit('splineFit', binCenters, np.abs(values), values, None, ampName)
+                # Anchor the spline to have zero correction at zero
+                # flux as well as at the lowest measured flux bin.
+                if binCenters[0] != 0.0:
```

Comment:
Should this check if any binCenters are negative?

Reply:
There shouldn't be any negative binCenters, as these are measured from a series of flat field images.  I've added a `raise` if there are negative binCenters, as that means something has gone wrong.